src/main/resources/application.yaml

```diff
@@ -26,7 +26,7 @@ cors:
   allowed_origins:
     - "https://familie-ef-soknad.nais.oera.no"
     - "https://www.nav.no/familie/alene-med-barn/soknad"
-    - "https://www.nav.no"
+    - "https://familie-ef-soknad-api.nais.oera.no"
```

Comment:
Hvis planen er at denne skal kalles fra api-et deres, kan kanskje ingressen for frontenden og nav.no fjernes?

Reply:
Ja, godt poeng, jeg var også usikker på dette. FOreløpig laster frontend opp direkte til _familie-dokument_, men mulig det er ryddigere om dette også går via api, for færrest mulig punkter å forholde seg til frontend. Det kan evt tas i egen oppgave :)

Reply:
Kan det ikke bare proxies gjennom nodejs appen? Da går det "bak" appen, men ikke gjennom apiet.